pkgs/applications/graphics/sane/config.nix

```diff
@@ -35,7 +35,9 @@ installSanePath = path: ''
 in
 stdenv.mkDerivation {
   name = "sane-config";
-  phases = "installPhase";
+  dontUnpack = true;
+  dontPatch = true;
```

Comment:
Why are the patch and fixup phases disabled? There should be no harm in keeping those, no? Fixup in particular seems important to run.

Reply:
i thought, that everything should stay the same as before and disable them. But i am in favor of enabling them.